web/packages/core/src/ruffle-player.ts

```diff
@@ -121,7 +121,7 @@ export class RufflePlayer extends HTMLElement {
     private container: HTMLElement;
     private playButton: HTMLElement;
     private unmuteOverlay: HTMLElement;
-    private preloader: HTMLElement;
+    private splashScreen: HTMLElement;
```

Comment:
While here: I think most (if not all) fields can be marked as `readonly`. Let's do it.